test/geometry/test_spatial_softargmax.py

```diff
@@ -74,57 +74,54 @@ def test_smoke_batch(self, device):
         m = kornia.SpatialSoftArgmax2d()
         assert m(input).shape == (2, 1, 2)
 
-    @pytest.mark.skip()
-    def test_top_left(self, device):
+    def test_top_left_normalized(self, device):
         input = torch.zeros(1, 1, 2, 3).to(device)
-        input[..., 0, 0] = 10.
+        input[..., 0, 0] = 1e16
```

Comment:
just curious, does it have to be that huge ?

Reply:
Probably not, no. That's just the first value that I tried.